common/lib/dependabot/clients/bitbucket.rb

```diff
@@ -52,6 +68,92 @@ def fetch_file_contents(repo, commit, path)
         response.body
       end
 
+      def commits(repo, branch_name = nil)
```

Comment:
I copied this most of the new methods from the Azure client, but I had to add the `repo` parameter to all of them.

Is this supposed to be using `source.repo` internally for all these parameters?